plugins/inputs/ravendb/README.md

```diff
@@ -0,0 +1,220 @@
+    - memory_allocated_in_mb  
+    - memory_installed_in_mb
+    - memory_low_memory_severity
+      - 0 -> None
+      - 1 -> Low
+      - 2 -> Extremely Low
+    - memory_physical_in_mb
+    - memory_total_dirty_in_mb
+    - memory_total_swap_size_in_mb
+    - memory_total_swap_usage_in_mb
+    - memory_working_set_swap_usage_in_mb
+    - network_concurrent_requests_count
+    - network_last_authorized_non_cluster_admin_request_time_in_sec (optional)
+    - network_last_request_time_in_sec (optional)
+    - network_requests_per_sec
+    - network_tcp_active_connections
+    - network_total_requests
+    - server_full_version
+    - server_process_id
+    - server_version
+    - uptime_in_sec
+  
+- ravendb_databases
+  - tags:
+    - url
+    - database_name
+    - database_id
+    - node_tag
+    - public_server_url (optional)
+  - fields:
+    - counts_alerts
+    - counts_attachments
+    - counts_documents
+    - counts_performance_hints
+    - counts_rehabs
+    - counts_replication_factor
+    - counts_revisions
+    - counts_unique_attachments
+    - statistics_doc_puts_per_sec
+    - statistics_map_index_indexes_per_sec
+    - statistics_map_reduce_index_mapped_per_sec
+    - statistics_map_reduce_index_reduced_per_sec
+    - statistics_request_average_duration
+    - statistics_requests_count
+    - statistics_requests_per_sec
+    - indexes_auto_count
+    - indexes_count
+    - indexes_disabled_count
+    - indexes_errors_count
+    - indexes_errored_count
+    - indexes_idle_count
+    - indexes_stale_count
+    - indexes_static_count
+    - storage_documents_allocated_data_file_in_mb
+    - storage_documents_used_data_file_in_mb
+    - storage_indexes_allocated_data_file_in_mb
+    - storage_indexes_used_data_file_in_mb
+    - storage_total_allocated_storage_file_in_mb
+    - storage_total_free_space_in_mb
+    - time_since_last_backup_in_sec (optional)
+    - uptime_in_sec
+
+- ravendb_indexes
+  - tags: 
+    - database_name
+    - index_name
+    - node_tag
+    - public_server_url (optional)
+    - url
+  - fields
+    - errors
+    - is_invalid
+    - lock_mode
+      - Unlock
+      - LockedIgnore
+      - LockedError
+    - mapped_per_sec
+    - priority
+      - Low
+      - Normal
+      - High
+    - reduced_per_sec
+    - state
+      - Normal
+      - Disabled
+      - Idle
+      - Error
+    - status
+      - Running
+      - Paused
+      - Disabled
+    - time_since_last_indexing_in_sec (optional)
+    - time_since_last_query_in_sec (optional)
+    - type
+      - None
+      - AutoMap
+      - AutoMapReduce
+      - Map
+      - MapReduce
+      - Faulty
+      - JavaScriptMap
+      - JavaScriptMapReduce
+
+- ravendb_collections
+  - tags:
+    - collection_name
+    - database_name
+    - node_tag
+    - public_server_url (optional)
+    - url
+  - fields
+    - documents_count
+    - documents_size_in_bytes
+    - revisions_size_in_bytes
+    - tombstones_size_in_bytes
+    - total_size_in_bytes
+
```

Comment:
It would be nice to show an example output in line-protocol format for this plugin. Only two lines or so for the user to get an idea.

Reply:
sure - added

Reply:
Thx, please mark as resolved.